backend/station_manager/app/src/main/java/com/StationManager/app/domain/train_station/Hall.java

```diff
@@ -8,6 +8,7 @@
 import java.util.stream.Collectors;
 
 public class Hall {
+    private Integer id;
```

Comment:
should also probably be final? it's only set by DB, right?

Reply:
@serozhenka do you mean it won't be used at all? we need to setup it somehow to distinguish halls and have references to other objects. What I think may be done here is ID to be added to constructor. What do you think?

Reply:
I don't say it won't be used at all. But you can make it final as it's set only once and also add it to the constructor